.github/workflows/kong-license.yaml

```diff
@@ -0,0 +1,29 @@
+name: Update Kong license secret
+
+on:
+  schedule:
+    - cron: '0 3 * * *'
+  workflow_dispatch: {}
+
+jobs:
+  test:
+    env:
+      PULP_USERNAME: ${{ secrets.PULP_USERNAME}}
+      PULP_PASSWORD: ${{ secrets.PULP_PASSWORD}}
+      PULP_LICENSE_URL: ${{ secrets.PULP_LICENSE_URL}}
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: License
+        id: license
+        run: |
+          LICENSE=$(curl -s -L -u"$PULP_USERNAME:$PULP_PASSWORD" "$PULP_LICENSE_URL")
+          echo "::add-mask::$LICENSE"
+          echo "::set-output name=LICENSE::$LICENSE"
+
+      - uses: gliech/create-github-secret-action@v1
```

Comment:
I'm a little iffy about this dependency. I'm not saying no, but I'm curious how hard it would be to put a simple curl one liner to update the license instead of using this dep?

Reply:
It would better for sure but it seems you need to

- get the key used for secrets encryption
- use the key to encrypt the secret contents
- call [github API](https://docs.github.com/en/rest/actions/secrets#create-or-update-a-repository-secret) with said encrypted contents

It's not rocket science but not-trivial either.

ref: https://blog.knoldus.com/playing-with-github-secret-api/#2-get-an-organization-public-key

Reply:
I gotcha. @rainest you got any thoughts on this one?

Reply:
Sounds good, let's pin this and I would consider this comment resolved.

Reply:
Done, PTAL at the updated version.